apps/frontend/src/pages/common-layout/common-layout.tsx

```diff
@@ -0,0 +1,27 @@
+import { type ReactNode } from "react";
+
+import { Header } from "~/libs/components/components.js";
+
+import styles from "./styles.module.css";
+
+type CommonLayoutProperties = {
+	children: ReactNode;
+};
+
+const CommonLayout = ({ children }: CommonLayoutProperties): JSX.Element => {
+	return (
+		<div className={styles["container"]}>
+			<Header />
+			<div className={styles["container__sidebar-content"]}>
+				<div
+					style={{ backgroundColor: "white", height: "100vh", width: "280px" }}
```

Comment:
try not to use vh unit if possible (better % or flex)

Reply:
These styles will then be removed when the header and sidebar are added